test/src/e2e_vm_tests/test_programs/should_pass/language/intrinsics/dbg_release/stdout.snap

```diff
@@ -19,17 +19,17 @@ ecal $r1 $r0 $zero $zero      ; ecal id fd zero zero
 ecal $r4 $r0 $r1 $r3          ; ecal id fd buf count
 
 > forc build --path test/src/e2e_vm_tests/test_programs/should_pass/language/intrinsics/dbg_release --release --asm final | sub ecal
-ecal $r4 $r5 $r2 $r3          ; ecal id fd buf count
```

Comment:
Just a curious remark how `__dbg` `ecal`s in `release` mode are "sensitive" to seemingly unrelated IR changes. The same effect we had when the inlining threshold was changed from 4 to 12 instructions.